src/java/org/apache/cassandra/io/sstable/SSTableHeaderFix.java

```diff
@@ -332,6 +347,16 @@ private void processSSTable(Descriptor desc)
             return;
         }
 
+        boolean hasDroppedMultiCellColumns = false;
+        for (DroppedColumn droppedColumn : tableMetadata.droppedColumns.values())
```

Comment:
need to fix `SSTableHeaderFixTest`, and all the tests just assume changes are made, and that's no longer (always) the case…

Reply:
done